### PR TITLE
Switch to test helper more resources operations

### PR DESCRIPTION
**Pull request by bogdando**
Use database, keystone, transport  basic test ops
from test helper common lib.

Depends-On: https://github.com/openstack-k8s-operators/lib-common/pull/171

Signed-off-by: Bohdan Dobrelia <bdobreli@redhat.com>

### CODE REVIEW

**Comment by gibizer** [comment]
I think you need to put the `replace` line into the go.mod file instead of the go.work file to get the CI pass

**Comment by bogdando** [comment]
so this probably depends on https://github.com/openstack-k8s-operators/nova-operator/pull/265 and will need a rebase after that

**Comment by bogdando** [comment]
this should pass now as I made it using the lib-common@c8a2b321 which doesn't bring in the most recent changes

File: test/functional/novaapi_controller_test.go [hunk 1]
```diff
@@ -332,16 +332,17 @@ var _ = Describe("NovaAPI controller", func() {
 				condition.ExposeServiceReadyCondition,
 				corev1.ConditionTrue,
 			)
-			GetService(types.NamespacedName{Namespace: namespace, Name: "nova-public"})
```
Comment (bogdando) [inline test/functional/novaapi_controller_test.go:335]: ouch, those changes look unexpeced, hence the test failure

**Review by gibizer — APPROVED** [review]
Looks good. Thanks @bogdando !

**Pull request merged**